Header header
string location_str
string date_time_str
string[] alert_classes_list
